test/test.js

```diff
@@ -20,6 +20,17 @@ const depsByVersion = versions.map((version) => {
 })
 
 describe.each(depsByVersion)('Chunk implementation for minecraft %s', (version, Chunk, Block) => {
+  const isPostFlattening = version.startsWith('1.13') || version.startsWith('1.14') ||
+    version.startsWith('1.15') || version.startsWith('1.16') || version.startsWith('1.17')
+
+  const serializesLightingDataSeparately = version.startsWith('1.14') || version.startsWith('1.15') ||
+    version.startsWith('1.16') || version.startsWith('1.17')
+
+  const newLightingDataFormat = version.startsWith('1.17')
+
+  const serializesBiomesSeparately = version.startsWith('1.15') || version.startsWith('1.16') ||
+    version.startsWith('1.17')
```

Comment:
Should these be inverted wherever used and renamed e.g. to isPreFlattening? The pre-flattening versions will always remain the same so that won't need edits, but every time there's a new release it will be a post-flattening version and this will need to be updated or else risk buggy behavior.
It might make more sense to special case the "old behavior" on all of these: isPreFlattening, doesNotSerializeLightingDataSeparately, pre1dot17lightingDataFormat, doesNotSerializeBiomesSeparately, etc.